templates/project-ts/src/interact.ts

```diff
@@ -84,13 +84,13 @@ try {
 } catch (err) {
   console.log(err);
 }
-if (sentTx?.hash() !== undefined) {
+if (sentTx?.hash !== undefined) {
```

Comment:
One thing to note, this check will never be undefined now since we compute the hash instead of getting it from the response. Could we change this check to be something else? We could check the errors array on the object instead.